chips/nrf52/src/uart.rs

```diff
@@ -121,6 +120,10 @@ impl UART {
         let regs = unsafe { &*self.regs };
         let tx = regs.event_endtx.get() != 0;
 
+        if regs.event_error.get() == 1 {
+            panic!("uart error event\r\n");
```

Comment:
Remove it completely, I think.

Re the PR breaking make sure you're soing everything completely from scratch including recompiling the apps. A bunch of small stuff has changed, not just that PR